flocx_market/objects/bid.py

```diff
@@ -0,0 +1,58 @@
+import flocx_market.db.sqlalchemy.api as db
+from flocx_market.objects import base
+from flocx_market.objects import fields
+
+
+class Bid(base.MarketPlaceObject):
+
+    fields = {
+        'marketplace_bid_id': fields.StringField(),
+        'creator_bid_id': fields.StringField(),
+        'creator_id': fields.StringField(),
+        'server_quantity': fields.IntegerField(),
+        'start_time': fields.DateTimeField(nullable=True),
+        'end_time': fields.DateTimeField(nullable=True),
+        'duration': fields.IntegerField(),
+        'status': fields.StringField(),
+        'server_config_query': fields.FlexibleDictField(nullable=True),
+        'cost': fields.FloatField()
+    }
+
+    @classmethod
+    def create(cls, data):
+        bid = db.bid_create(data)
+        return bid.to_dict()
+
+    @classmethod
+    def get(cls, bid_id):
+        if bid_id is None:
+            return cls.get_all()
```

Comment:
I don't know that we want to return all the bids if bid_is is None; I think we just throw an error